src/kaleidoscope/plugin/WinKeyToggle.h

```diff
@@ -17,7 +17,14 @@
 
 #pragma once
 
-#include "kaleidoscope/Runtime.h"
+#include <stdint.h>                             // for uint8_t
+#include "kaleidoscope/KeyAddr.h"               // for KeyAddr
+#include "kaleidoscope/event_handler_result.h"  // for EventHandlerResult
+#include "kaleidoscope/plugin.h"                // for Plugin
+
+namespace kaleidoscope {
+class Key;
+}
```

Comment:
Do we really not offer a header which can provide this forward declaration? (I presume that's what's going on)

Reply:
Forward declarations are a way to avoid includes. That's why iwyu suggests them.